docs/administration.customers.md

```diff
@@ -59,7 +59,8 @@ If you want to prevent a customer from creating new [meshProjects](meshcloud.pro
 
 ### Access managed Customer Accounts
 
-As a partner user you can assign yourself a role on managed managed customer accounts.
+As a partner user you can assign yourself a role on managed customer accounts. Please note that this only works
```

Comment:
```suggestion
As a partner user you can assign yourself a role on managed customer accounts. Please note that the role you receive in the customer via the "add myself" button is depending on the configuration of the meshStack. You have the option to specify if the role "customer admin" or "customer employee" is granted.
```

Reply:
I still miss the info at this part that it is up to configuration

Reply:
Updated.